mne/epochs.py

```diff
@@ -775,6 +775,46 @@ def drop_log_stats(self, ignore=['IGNORED']):
         """
         return _drop_log_stats(self.drop_log, ignore)
 
+    def apply_reject(self, reject=None, flat=None):
```

Comment:
On Tue, Mar 24, 2015 at 5:51 PM, Alexandre Gramfort <
notifications@github.com> wrote:

> In mne/epochs.py
> <https://github.com/mne-tools/mne-python/pull/1900#discussion_r27048149>:
>
> > @@ -775,6 +775,46 @@ def drop_log_stats(self, ignore=['IGNORED']):
> >          """
> >          return _drop_log_stats(self.drop_log, ignore)
> >
> > +    def apply_reject(self, reject=None, flat=None):
>
> thinking about this why not adding the reject and flat to drop_bad_epochs
> method?
>
done, I agree it's cleaner that way. Tests pass on my local box. Your wish
is my command :)


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/1900/files#r27048149>.
>